x-pack/metricbeat/module/googlecloud/stackdriver/metricset.go

```diff
@@ -212,13 +219,15 @@ func (m *MetricSet) metricDescriptor(ctx context.Context, client *monitoring.Met
 		for _, mt := range sdc.MetricTypes {
 			req := &monitoringpb.ListMetricDescriptorsRequest{
 				Name:   "projects/" + m.config.ProjectID,
-				Filter: fmt.Sprintf(`metric.type = "%s"`, mt),
+				Filter: fmt.Sprintf(`metric.type = "%s"`, sdc.ServiceName+".googleapis.com/"+mt),
```

Comment:
Is this correct for all services? Can the path be constructed the same way from the service name?

Also, can it happen that a service has metrics with different paths?

Reply:
I believe this is correct for all services. From the [google cloud documentation](https://cloud.google.com/monitoring/api/metrics_gcp#gcp-compute), it shows under each service, `The following metric type names must be prefixed with xxx.googleapis.com/. That prefix has been omitted from the entries in the table.`

Reply:
Sounds good then :+1: